go.mod

```diff
@@ -72,6 +72,7 @@ require (
 	github.com/exponent-io/jsonpath v0.0.0-20210407135951-1de76d718b3f // indirect
 	github.com/fatih/color v1.17.0 // indirect
 	github.com/felixge/httpsnoop v1.0.4 // indirect
+	github.com/foxcpp/go-mockdns v1.1.0 // indirect
```

Comment:
It feels like this is not strictly related to bumping `github.com/kuadrant/authorino`, but I guess it's OK?